1_space_odyssey/js/mannuel-9000/app.js

```diff
@@ -113,11 +113,16 @@ console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, ha
     a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid 
        cannot go outside, console log the correct log based on that condition. 
 */
-
+var cangooutside = true;
+console.log(cangooutside);
 /* 
  uncomment these console logs and write your if else statement and use them to console log out the correct statement
-
  console.log("JetPack is enabled!!! off I go outside to fix the satellite");
  console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
-
- */
\ No newline at end of file
+ */
+if (hasJetPack){
+  console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+}
+else if (hasJetPack){
```

Comment:
do not need to use if...else if.
rather only use if..else.

if you are only checking one condition or one boolean value, if...else is good enough.

```JavaScript
if (someThing) {
  ...
} else {
 ...
}
```

if you are checking more than one condition then you can use if...else if...else if, and then if there are too many use a switch statement.

![https://media.giphy.com/media/3o85xqzprB9DhZmVk4/giphy.gif](https://media.giphy.com/media/3o85xqzprB9DhZmVk4/giphy.gif)

Reply:
I only just realised you have edited my file `/mannuel-9000/app.js`

please create a folder for yourself of `tina-9000/app.js` and push it up and I will then merge it.